### PR TITLE
CCI17 Add timeout flow

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**
Build the timeout flow. 
Deliver an SMS to the user on first timeout.

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps please review

File: src/ussd/app.js [hunk 1]
```diff
@@ -23,15 +37,99 @@ go.app = function() {
...
+                });
+        };
+
+        self.dial_back = function(e) {
+            if (!self.should_send_dialback(e)) { return; }
+            return self.send_dialback();
+        };
+
+        self.get_dialback_sms = function() {
+            return $("Thanks for using the *120*662# MMC service! Dial back "
```
Comment (imsickofmaps) [inline src/ussd/app.js]: `*120*662#` should be context driven

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps please review...

**Comment by imsickofmaps** [comment]
👍

**Pull request merged**